### PR TITLE
Angular version upgrade

### PR DESCRIPTION
**Pull request by shkurata**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,29 +12,29 @@
...
+    "zone.js": "~0.10.2"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.10.0",
-    "@angular-devkit/build-ng-packagr": "~0.10.0",
-    "@angular/cli": "^7.3.9",
-    "@angular/compiler-cli": "~7.2.0",
-    "@angular/language-service": "~7.2.0",
+    "@angular-devkit/build-angular": "~0.900.5",
+    "@angular-devkit/build-ng-packagr": "~0.900.5",
```
Comment (jbrems) [inline package.json]: This looks wrong

File: projects/ng-wizard/package.json [hunk 2]
```diff
@@ -1,23 +1,29 @@
 {
   "name": "@cmdap/ng-wizard",
-  "version": "1.8.0",
+  "version": "2.0.0",
```
Comment (jbrems) [inline projects/ng-wizard/package.json]: Make it version 9.0.0

File: projects/ng-wizard/package.json [hunk 3]
```diff
@@ -1,23 +1,29 @@
...
   "description": "A simple wizard/stepper component for Angular 7 utilizing Angular Routing for navigation.",
```
Comment (jbrems) [inline projects/ng-wizard/package.json]: Update the description

File: projects/ng-wizard/tsconfig.lib.prod.json [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "./tsconfig.lib.json",
+  "angularCompilerOptions": {
+    "enableIvy": false
```
Comment (jbrems) [inline projects/ng-wizard/tsconfig.lib.prod.json]: No ivy? That is like the big thing in Angular 9...

File: package.json (hunk 1 above)
Comment (shkurata) [reply package.json]: That is the versioning that the authors of the package are using. [@angular-devkit/build-angular](https://www.npmjs.com/package/@angular-devkit/build-angular).

File: projects/ng-wizard/tsconfig.lib.prod.json (hunk 4 above)
Comment (shkurata) [reply projects/ng-wizard/tsconfig.lib.prod.json]: It is not recommended to use Ivy when publishing as npm. [angular.io](https://angular.io/guide/creating-libraries#publishing-your-library)

**Pull request merged**